### PR TITLE
hERG drug interaction predictor

### PR DESCRIPTION
**Pull request by Nithanaroy**
Define a ML pipeline which predicts if a drug interacts / blocks the functioning of hERG gene or not. The pipeline includes,
- data pulled from TDC
- data preparation using DeepPurpose
- train a binary classifier using DeepPurpose
- tune the model using Ray
- log the results to mlflow (similar to weights and biases)

The change also includes the required conda environment to run this example. Eventually we will publish the best model to TDC Huggingface hub.

## Appendix
Screen Shot 2022-11-27 at 3 42 04 PM
**Figure 1:** example mlflow experiment exported by the code showing 3 values of hidden layers of GNN explored

localhost_5001_
**Figure 2:** example mlflow trail of the above experiment showing all the fields for each experiment

### CODE REVIEW

File: examples/huggingface_examples/herg/hERG_inhib.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
```
Comment (Nithanaroy) [inline examples/huggingface_examples/herg/hERG_inhib.py:1]: auto generated file using `nbconvert` suitable for reviewing.

File: examples/huggingface_examples/herg/hERG_inhib.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+from DeepPurpose import utils as dp_utils, CompoundPred
+
+
+# ## Prepare Data
+
+X, y = Tox(name = 'herg_central', label_name="hERG_inhib").get_data(format = 'DeepPurpose')
+
+
+X, y = X[:100], y[:100]
```
Comment (Nithanaroy) [inline examples/huggingface_examples/herg/hERG_inhib.py:23]: This is to quickly iterate over the code during development. This line will be removed while tuning the final model.

**Comment by Nithanaroy** [comment]
Next, I'll add the pipeline code to publish the best model after tuning to HF hub

File: examples/huggingface_examples/herg/hERG_inhib.py (hunk 2 above)
Comment (kexinhuang12345) [reply examples/huggingface_examples/herg/hERG_inhib.py:23]: Sounds good!

**Review by kexinhuang12345 — APPROVED** [review]

**Pull request merged**